app/views/appointment_summaries/preview.html.erb

```diff
@@ -16,3 +16,7 @@
     </div>
   </div>
 </div>
+
+<% content_for :body_end do %>
+  <%= javascript_include_tag 'preview' %>
```

Comment:
Better to include in the main application.js payload I think, its a very small piece of script (and is likely to remain so).